### PR TITLE
Add TI ina220 driver

### PR DESCRIPTION
**Pull request by benjinne**
## Describe problem solved by this pull request
Adds the INA220 driver

## Describe your solution
based on the INA226 driver

## Test data / coverage
Tested using an ina220 and validated current voltage readings were accurate compared to multi-meter

### CODE REVIEW

**Comment by benjinne** [comment]
Hey let me know what I need to do for this @dagar

**Comment by dirksavage88** [comment]
Are you using a custom PCB/prototype for this? Or an evm that TI sells? I'm trying to find one but only see ina226 boards (Ali & banggood)

**Comment by benjinne** [comment]
> Are you using a custom PCB/prototype for this? Or an evm that TI sells? I'm trying to find one but only see ina226 boards (Ali & banggood)

It's for a custom PDB I'm designing

**Review by AlexKlimaj — CHANGES_REQUESTED** [review]
Why change the drv_sensor values for the existing sensors?

**Comment by benjinne** [comment]
> Why change the drv_sensor values for the existing sensors?

There wasn't any room left to add more so I separated out the power_devtypes to

File: src/drivers/power_monitor/ina220/ina220.cpp [hunk 1]
```diff
@@ -0,0 +1,376 @@
+/****************************************************************************
+ *
+ *   Copyright (c) 2019-2020 PX4 Development Team. All rights reserved.
```
Comment (dagar) [inline src/drivers/power_monitor/ina220/ina220.cpp:3]:
```suggestion
 *   Copyright (c) 2022 PX4 Development Team. All rights reserved.
```

File: src/drivers/power_monitor/ina220/ina220.h [hunk 2]
```diff
@@ -0,0 +1,223 @@
+/****************************************************************************
+ *
+ *   Copyright (C) 2019 PX4 Development Team. All rights reserved.
```
Comment (dagar) [inline src/drivers/power_monitor/ina220/ina220.h:3]:
```suggestion
 *   Copyright (C) 2022 PX4 Development Team. All rights reserved.
```

File: src/drivers/power_monitor/ina220/ina220_main.cpp [hunk 3]
```diff
@@ -0,0 +1,145 @@
+/****************************************************************************
+ *
+ *   Copyright (C) 2021 PX4 Development Team. All rights reserved.
```
Comment (dagar) [inline src/drivers/power_monitor/ina220/ina220_main.cpp:3]:
```suggestion
 *   Copyright (C) 2022 PX4 Development Team. All rights reserved.
```

File: src/drivers/power_monitor/ina220/ina220_params.c [hunk 4]
```diff
@@ -0,0 +1,98 @@
+/****************************************************************************
+ *
+ *   Copyright (c) 2019, 2021 PX4 Development Team. All rights reserved.
```
Comment (dagar) [inline src/drivers/power_monitor/ina220/ina220_params.c:3]:
```suggestion
 *   Copyright (c) 2022 PX4 Development Team. All rights reserved.
```

**Review by dagar — APPROVED** [review]

**Comment by dagar** [comment]
Changing the sensor type is usually a bad idea, but in this case it's harmless.

**Pull request merged**